src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -463,7 +463,7 @@ module.exports = function<T, P, I, TI, C, CX, CI>(
     }
     const ref = finishedWork.ref;
     if (ref) {
-      const instance = getPublicInstance(finishedWork.stateNode);
+      const instance = (getPublicInstance(finishedWork.stateNode) : any);
```

Comment:
the flow type for `instance` on master is `any`. Whilst adding the `PI` parameter does end up saying `instance` is `PI`, that fails flow due to ref expecting:

```
ref: null | (((handle : ?Object) => void) & { _stringRef: ?string }),
```

I suspect this could be improved, but for time constraints at the moment I simply dropped it back to equal the current inference.